lib/themes/dosomething/paraneue_dosomething/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "start": "npm run clean && npm run modernizr && webpack",
+    "start": "check-dependencies & npm run clean && npm run modernizr && webpack",
```

Comment:
D'oh, I was too quick merging! 🐰 

Switching it to `&&` would halt the following steps from running if a dependency was out of date (whereas `&` will let it continue through even if `check-dependencies` returns an error code).